Formula/r/ratarmount.rb

```diff
@@ -12,6 +12,9 @@ class Ratarmount < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "11268d8d0bd4a24d02de87d4834083d228f6d4a1de4ee88a5686039fb6679ac4"
   end
 
+  # Doesn't support libfuse 3.16>
```

Comment:
should be as 3.16+?
```suggestion
  # Doesn't support libfuse 3.16+
```

also can you file an upstream issue about this as well?

seems pretty actively maintained https://github.com/mxmlnkn/ratarmount